$NetBSD: patch-CMakeLists.txt,v 1.3 2021/03/03 10:20:04 fox Exp $

Add new src folders to CMakeList.txt (should be fixed in the next upstream).
Only define __LINUX_ALSA__ on Linux.

--- CMakeLists.txt.orig	2021-03-01 19:04:28.000000000 +0000
+++ CMakeLists.txt
@@ -10,6 +10,8 @@ file(GLOB ft2-clone_SRC
     "${ft2-clone_SOURCE_DIR}/src/*.c"
     "${ft2-clone_SOURCE_DIR}/src/gfxdata/*.c"
     "${ft2-clone_SOURCE_DIR}/src/mixer/*.c"
+    "${ft2-clone_SOURCE_DIR}/src/modloaders/*.c"
+    "${ft2-clone_SOURCE_DIR}/src/smploaders/*.c"
 )
 
 add_executable(ft2-clone ${ft2-clone_SRC})
@@ -24,7 +26,9 @@ endif()
 
 target_link_libraries(ft2-clone
     PRIVATE m asound pthread ${SDL2_LIBRARIES})
-target_compile_definitions(ft2-clone PRIVATE __LINUX_ALSA__)
+if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")    
+    target_compile_definitions(ft2-clone PRIVATE __LINUX_ALSA__)
+endif()
 
 install(TARGETS ft2-clone
     RUNTIME DESTINATION bin )
